src/stream/src/executor/source/source_executor.rs

```diff
@@ -393,12 +404,11 @@ impl<S: StateStore> SourceExecutor<S> {
         );
 
         // Replace the source reader with a new one of the new state.
-        let (reader, _) = self
-            .build_stream_source_reader(source_desc, Some(target_state.clone()), false)
```

Comment:
It seems that the original `build_stream_source_reader` is not used anymore. May remove it.